stix_shifter_modules/splunk/stix_translation/json/to_stix_map.json

```diff
@@ -255,5 +255,79 @@
     "key": "email-message.is_multipart",
     "object": "email-message",
     "transformer": "StringToBool"
+  },
+  "source": [
+    {
+      "key": "x-splunk.log_source",
+      "object": "x-splunk"
+    }
+  ],
+  "_sourcetype": [
+    {
+      "key": "x-splunk.provider",
+      "object": "x-splunk"
+    }
+  ],
+  "DeviceType": [
+    {
+      "key": "x-splunk.device_type",
+      "object": "x-splunk"
+    }
+  ],
+  "Direction": [
+    {
+      "key": "x-splunk.direction",
+      "object": "x-splunk"
+    }
+  ],
+  "severity": [
+    {
+      "key": "x-splunk.severity",
+      "object": "x-splunk"
```

Comment:
redundant object. you don't need to put severity in `x-splunk.severity`. you can just use `x-ibm-finding.severity`

Reply:
okay. working on it.

Reply:
@mdazam1942  Resolved redundant object severity.